src/components/cardComponent.css

```diff
@@ -0,0 +1,31 @@
+#card{
```

Comment:
Try to keep the name of css file also similar to the component name (i.e. CardComponent.js will have it's css in CardComponent.css). while this will not provide any while in terms of operation of the app, it'll certainly make life easier for whoever comes in to work on your code